client-templates/github-com/accept.json.sample

```diff
@@ -959,6 +959,18 @@
       "path": "/:name/:repo/:path*%2FPodfile.lock",
       "origin": "https://${GITHUB_TOKEN}@${GITHUB_RAW}"
     },
+    {
+      "//": "used to scan Dockerfile",
+      "method": "GET",
+      "path": "/:name/:repo/:path*/Dockerfile",
+      "origin": "https://${GITHUB_TOKEN}@${GITHUB_RAW}"
+    },
+    {
+      "//": "used to scan Dockerfile",
+      "method": "GET",
+      "path": "/:name/:repo/:path*%2FDockerfile",
+      "origin": "https://${GITHUB_TOKEN}@${GITHUB_RAW}"
+    },
```

Comment:
This should have `API` as well as `RAW`

Reply:
From other endpoints, looks like the API ones are `"path": "/repos/:name/:repo/contents/:path*/.snyk",` and the RAW are `"path": "/:name/:repo/:path*/.snyk",`

Reply:
Good catch, thanks.